oe_theme.theme

```diff
@@ -771,3 +771,16 @@ function _oe_theme_preprocess_input_text(array &$element): void {
   // Map rest array elements.
   $element['ecl_array']['has_error'] = !empty($element['#errors']);
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function oe_theme_preprocess_pattern_footer(array &$variables): void {
+  $social_media_links = &$variables['corporate_footer']['social_media']['items'] ?? [];
+  // Workaround related to UI Pattern module feature
+  // which recognize array with "type" key element as renderable.
+  foreach ($social_media_links as $key => $link) {
```

Comment:
@sergepavle : Why you don't use the reference on the foreach?
```
foreach ($social_media_links as &$link) {
    $link['type'] = $link['link_type'];
    unset($link['link_type']);
}
```

Reply:
@voidtek I've checked and it seems we don't use links value objects anywhere. Anyway we decided not to implement this as a pattern since this is not a reusable theme component.